### PR TITLE
Feat/msg system webusb android

### PR DESCRIPTION
**Pull request by tomasklim**
## Description

- web usb android info
- prompt users in old versions to update app

## Related Issue

#6512

## Screenshots (if appropriate):
Screenshot 2022-10-13 at 17 19 41

### CODE REVIEW

**Review by sime — APPROVED** [review]
I know the 'x' is not ideal, but it will suffice.

**Review by mroz22 — APPROVED** [review]

File: packages/message-system/src/config/config.v1.json [hunk 1]
```diff
@@ -264,6 +264,40 @@
...
+            ],
+            "message": {
+                "id": "b6c6dbeb-dca2-4481-b421-0381d2bd4927",
+                "priority": 91,
+                "dismissible": true,
+                "variant": "warning",
+                "category": "banner",
+                "content": {
+                    "en-GB": "USB connections are currently unavailable for Chrome on Android. Please use the desktop app instead.",
+                    "en": "USB connections are currently unavailable for Chrome on Android. Please use the desktop app instead.",
```
Comment (matejkriz) [inline packages/message-system/src/config/config.v1.json:294]: what about
```suggestion
                    "en": "USB connections are unavailable for current Chrome on Android. Please use the desktop app instead.",
```
?

Comment (tomasklim) [reply packages/message-system/src/config/config.v1.json:294]: @sime

Comment (sime) [reply packages/message-system/src/config/config.v1.json:294]: Yes, and it matches the text we agreed upon in Slack :)

**Pull request merged**